.env.template

```
@@ -1,6 +1,9 @@
 # Supabase Configuration (REQUIRED)
+# The backend uses the SERVICE ROLE key (it performs privileged server-side
+# writes and bypasses RLS). RLS policies are a defense-in-depth backstop for any
+# direct anon/PostgREST access, not the backend's primary access control.
 SUPABASE_URL=your_supabase_project_url
-SUPABASE_KEY=your_supabase_anon_key
+SUPABASE_KEY=your_supabase_service_role_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)